tcpdialer.go

```diff
@@ -129,6 +129,8 @@ type TCPDialer struct {
 	// Changes made after the first Dial will not affect anything.
 	Concurrency int
 
+	LocalAddr *net.TCPAddr
```

Comment:
Can you please add a comment to explain this option.

Reply:
@erikdubbelboer 
this would give an option to chose from which IP we will dial later. When local address is nil, net.DialTCP will choose local address automatically. This option is to choose a public IP, which is useful, for example, on systems with multiple public IPs.

@mikefaraponov please document this option

Reply:
@kirillDanshin I know what it does 😄 if I didn't I would be a lousy maintainer. I mean he should add a comment as documentation.